packages/@lwc/integration-karma/test/component/properties/index.spec.js

```diff
@@ -7,6 +7,7 @@ import Component from 'x/component';
 const expectedEnumerableProps = [
     'accessKey',
     'addEventListener',
+    'attachInternals',
```

Comment:
These two lists come from the `LightningElement` which is why `attachInternals` is still visible here.